cquadpack/bld.bat

```diff
@@ -0,0 +1,15 @@
+if "%CMAKE_GENERATOR%" == "" (

```

Comment:
I don't like this too much. Why not make it:

    set CMAKE_GENERATOR=NMake Makefiles

?

I mean, I don't think there's anything to be gained by letting users to override CMAKE_GENERATOR from the environment.